### DISCUSSION TITLE
Show the Preview (viewport) Dimensions fields in the Editor Top Bar

### DISCUSSION THREAD

**Discussion by AndreasH08** [elementor/elementor#22432] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I miss the editor showing the current size of the screen, and I wish there were a better way to see the page I was working on in 4K and laptop size. 

### Describe the solution you'd like

It would be nice to see it added to the top bar experiment, and I'd love something similar to the inspect toggle device toolbar, making it easier to work on all screen sizes.

### Describe alternatives you've considered

The inspect function can give me all the info, but it would be awesome to see it inside the top bar experiment

**Comment by nicholaszein** [comment]
This request was referenced in the following issues and discussions:

**Discussions**
- #24404 
- #24713
- #31683

**Comment by maurocarreraok** [comment]
+1

**Comment by Darrenpaullittle** [comment]
https://discord.com/channels/1164474724626206720/1209513226987900978/threads/1384218392256450741
